config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -285,17 +284,40 @@ config MBEDTLS_ENABLE_HEAP
 
 config MBEDTLS_HEAP_SIZE
     default 8192
+    
+# Enable PSA Crypto dependencies for Matter
 
-config NRF_SECURITY_ADVANCED
+config CHIP_CRYPTO_PSA
+    default n
+    select EXPERIMENTAL
```

Comment:
Add this to the original CHIP_CRYPTO_PSA definition and remove this one?

Reply:
The original Kconfig is defined in Zephyr's config space, so it will impact Telink also if we go upstream with it. Do you think it is ok?

Reply:
I requested removing it from there :) It's because I thought it's not ok to select EXPERIMENTAL for all Zephyr platforms, while only we (nrf platform) consider it experimental. I suggested something different here: https://github.com/nrfconnect/sdk-connectedhomeip/pull/311#discussion_r1432565531

Reply:
Oh, I see, let's leave it as it is, but `default n` is redundant, so it can be removed at least :)

Reply:
But maybe I care about the others too much and we should just set it in the Zephyr, as it was.

Reply:
Ok, discussed with the Team, that we're removing experimental at all.